README.md

```diff
@@ -1,33 +1,37 @@
-# Testgrid Jenkins Shared Library
+# Testgrid Jenkins Library
 
-Library that hosts Testgrid shared components and libraries. This contain a bunch of Jenkins pipeline critical for testgrid work:
+This repo contains all the pipeline code that is being used in the Jenkins TestGrid Pipeline.
 
-* [Testgrid job execution pipeline](https://github.com/wso2-incubator/testgrid-jenkins-library/blob/00ee9f2e666119d547ace504f88ce74d6ba97aa1/vars/Pipeline.groovy)
-* [GitOps based testgrid job creation pipeline](https://github.com/wso2-incubator/testgrid-jenkins-library/blob/15f7cfbb9e54f09aa859adb505ff64f535d6d20e/vars/JobCreatorPipeline.groovy)
+## Jenkins version, blueocean version
 
-Following is the current folder hierarchy of the source. This structure is required for the Jenkins shared library.
+Jenkins:- Jenkins 2.319.3
+Blueocean Version:- 1.25.3
 
-````
-.
-├── README.md
-├── pom.xml
-├── src
-│   └── org
-│       └── wso2
-│           └── tg
-│               └── jenkins
-│                   ├── Log.groovy
-│                   ├── alert
-│                   │   ├── Email.groovy
-│                   │   └── Slack.groovy
-│                   ├── executors
-│                   │   └── TestExecutor.groovy
-│                   └── util
-│                       ├── AWSUtils.groovy
-│                       └── Common.groovy
-├── test
-│   └── UtilTest.groovy
-└── vars
-    ├── Pipeline.groovy <- testgrid job execution pipeline
-    └── JobCreatorPipeline.groovy <- testgrid job creator via gitops
-````
+## Testgrid Jenkins Pipeline Variables
+
+1. **product**- The WSO2 product that needs to be tested from TestGrid.
+Allowed Values- apim, is
+2. **cfn_repo_url**- The WSO2 product CFN repo name.
+Allowed Value Format- <https://github.com/janethavi/aws-apim.git>
+3. **product_version**- The product version that needs to be tested using testgrid.
+Allowed Value Format- 3.2.0
+4. **product_deployment_region**- The region where the product stack is getting deployed.
+Allowed Values- us-east-2, 
```

Comment:
Please remove the comma at the end of line or add other regions of there are any.